src/content/docs/tutorial-create-alerts/create-an-alert.mdx

```diff
@@ -13,37 +17,148 @@ import alertsThresholds from 'images/alerts_screenshot-crop_set-thresholds.webp'
+
+import alertsGoldenEntities from 'images/alerts_screenshot-crop_golden-metric-entities.webp'
+
+You can create an alert condition in New Relic using 1 of 3 options: using a chart, using a NRQL query, or using golden metrics. While the beginning of the creation process is different depending on which option you choose, all the subsequent steps are the same. This means that once you've created a condition one way, the experience you gain from doing so will transfer over to others that you create using either of the other options.
+
+For this tutorial, we've included steps for all 3 of the aforementioned options, so you can get started with your first condition any way you want to. Just select the path you want to follow in the first step, and by the end of the procedure you'll have successfully made your first alert condition using New Relic!
 
 <Callout variant="tip">
 If you need more information on any of the options or terms covered in the steps below, check out our [alerts docs](/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/basic-alerting-concepts/).
     </Callout>
 
 <Steps>
     <Step>
-## Select a chart and define your signal
-
-<img
+<Tabs>
+	<TabsBar>
+        <TabsBarItem id="1">
+            Create using a chart
+        </TabsBarItem>
+        <TabsBarItem id="2">
+            Create using the guided mode
+        </TabsBarItem>
+        <TabsBarItem id="3">
+            Create using a NRQL query
+        </TabsBarItem>
+    </TabsBar>
+
+    <TabsPages>
+        <TabsPageItem id="1">
+<SideBySide>
+    <Side>
+Starting from a chart is one of the easiest ways to set up a New Relic alert condition. Instead of creating a NRQL query from scratch, charts let you use pre-existing NRQL queries to begin the process. 
+    
+In this example, we'll set up an alert condition using an error response chart. Follow the steps below with any type of chart you like to begin.
+    </Side>
+    <Side>
+    <img
+    title="Create chart alert"
+    alt="A screenshot displaying where to create an alert condition from a chart in New Relic"
+    src={alertsCreate}
+/>
+    </Side>
+</SideBySide>
+<SideBySide>
+    <Side>
+    <img
     title="Defining a signal"
     alt="A screenshot detailing how to select a query for an alert condition"
     src={alertsSignal}
-/>
+    />
+    </Side>
+    <Side>
+1. Go to any dashboard. Then, find a chart you want to use, click the "**...**" menu, and select **Create alert condition**.
+
+2. Select the signal, or stream of data, you want to use to create the alert condition. To keep things simple, **Build a query** is the default option for conditions created this way.
+
+3. Modify the NRQL query to match what you want you to monitor, if necessary. For now, we recommend leaving the query as is, but once you have a better handle on alerts and NRQL queries, you can change the query to monitor nearly any signal.
```

Comment:
this is good information. do you think there's space to describe what the NRQL query translates into in English terms?

Reply:
I moved the callout to earlier in the procedure to provide more context for NRQL and present opportunities for learning about it to before the reader commits to the steps in the procedure. Thanks for the feedback!